src/test/mocks/store.ts

```diff
@@ -117,6 +121,7 @@ export const mockStore: ReduxState = {
     showConsentBanner: true,
     showSnapsModal: true,
     snapInstalled: false,
+    watchedAssets: [btcAssetId, ethAssetId, usdtAssetId, usdcAssetId, dogeAssetId, bchAssetId],
```

Comment:
Unless there's unit tests added for reducers/selectors, we don't need these

```suggestion
    watchedAssets: [],
```

Reply:
@gomesalexandre I was gonna ask you about this actually. I'd like to load up the watched assets with a default list so the user isn't really greeted with a blank list on first run. What is the best way to do that?